feat(unlock-protocol-com) - members page blogpost (#9571)
unlock-protocol/unlock: unlock-protocol-com/blog/new-filters-members-page.md

---

> title: New Members page Filters

Reviewer: ```suggestion
title: Members page Filters
```

> subTitle: A new way to filter for keys in members page

Reviewer: ```suggestion
subTitle: A new way to filter keys in the members page
```

authorName: Kalidou Diagne
publishDate: September 8, 2022
description: Unlock introduce a new set of filters to improve search experience.
image: /images/blog/new-filters-members-page/filter-overview.png
---

> Our Members page is now improved with more specific filters, that now makes more simplier to find a specific key.

Reviewer: ```suggestion
Our Members page is now improved with more specific filters, which make finding specific memberships much easier.
```

> The set of filter introduced are `owner`, `toked id`, `check in time`, `type of expiration` and `email`.

Reviewer: ```suggestion
We have introduced filters that let you find specific memberships by `owner`, `toked id`, `check-in time`, `type of expiration`, and `email`.
```

### Filter selection

![Filter selection](/images/blog/new-filters-members-page/filter-selection.png)

> When a filter is selected you will be able to see a input or a dropdown (with a set of default values) based on the active filter.

Reviewer: ```suggestion
When a filter is selected the user will see an input or a dropdown (with a set of default values) that lets them enter the specific values they are looking for.
```

![Filter by owner](/images/blog/new-filters-members-page/filter-by-owner.png)

### Filters in details

#### Owner

> This filter refers to public address, you will be able to filter by partial text or also with ens domain.

Reviewer: ```suggestion
This filter refers to the public wallet address, the user will be able to filter by partial (`0x123` to find all addresses that start with `0x123`) text or also with an [ENS domain](https://ens.domains/).
```

![Filter by owner](/images/blog/new-filters-members-page/filter-by-owner-partial.png)

#### Token id

> This filter refers to token id, you will be able to filter by a specific token id.

Reviewer: ```suggestion
This filter refers to token id, the user will be able to filter by a specific token id.
```

![Filter by owner token id](/images/blog/new-filters-members-page/filter-by-token-id.png)

#### Check in time

> This filter shows all keys marked as checked in.

Reviewer: ```suggestion
This filter shows all keys marked as checked in. This is useful for conferences that are trying to list all ticket holders that did show up and attended the event.
```

_Only lock manager are able to see this filter because checked in time is a protected data._

![Filter by checked in](/images/blog/new-filters-members-page/filter-checked-in.png)

#### Email

> This filter refers to email, and gives the ability to search a key by email if present in the metadata.

Reviewer: ```suggestion
This filter refers to email if it was [collected during checkout](https://docs.unlock-protocol.com/tools/checkout/collecting-metadata), and gives the ability to search a key by email if present in the metadata.
```

> _Only lock manager are able to see this filter because email is a protected data._

Reviewer: ```suggestion
_Only lock managers are able to see this filter because email is protected data._
```
